IO/GeoFileParser.cpp

```diff
@@ -496,7 +496,7 @@ bool GeoFileParser::parseDoorNode(TiXmlElement * xDoor, int id, Building* buildi
      }
      //----------------- dn
      int DN = xmltof(xDoor->Attribute("dn"), -1.0);
-     if(DN >= 0)
+     if(DN > 0)
```

Comment:
With this, it is not possible to start the counting right at the beginning of the simulation, right? For example if I want to let 5 peds through a door: dn = 0 and max_agents = 5 won’t work

Reply:
The decisive condition is in this [line](https://github.com/JuPedSim/jpscore/blob/develop/Simulation.cpp#L1112)

```
if( trans->GetPartialDoorUsage() == trans->GetDN() )
```
`PartialDoorUsage is > 0` because in the [line before](https://github.com/JuPedSim/jpscore/blob/develop/Simulation.cpp#L1106)
it's incremented by 1.

So `dn>0` should be satisfied otherwise that if will not be entered.

Reply:
Basically my question is, if it will be possible to start the flow regulation with the beginning of the simulation or only if x>=1 peds have passed the door?

Reply:
The regulation of a transition kicks in once at least 1 pedestrian passes that transition. 

The reason why we introduced `dn` was to avoid too many updates.  

So, a reasonable value for `dn` should definitely depend on the simulation size.

`dn=0` makes no sense in this regard. `dn=1`could lead to too frequent updates.

Maybe, it would be more user friendly if we remove this parameter and decide for a default value. But, then we could lose some flexibility too.

Reply:
Just for my understanding: If I set `dn=5` and `max_agents=5`, will the door closed after 5 or 10 agents passed it?

Reply:
In that case, 5 pedestrians will pass the door. 
```
if( trans->GetPartialDoorUsage() == trans->GetDN() )

```

is true 

--> Regulation starts. 
But then `max_agents=5` 
--> door closes permanently!

Reply:
Ok, then I understood that part wrong.